test/react/firmware-update/modal.test.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react';
+
+import * as mobx from 'mobx';
+
+import DownloadBackupModalState from '../../../app/scripts/react-directives/firmware-update/modal';
+
+it('constructor without id', () => {
```

Comment:
Тут же тебе тестовый фреймворк подсказывает, как предложение строить :)
```suggestion
it('may be constructed without id', () => {
```

При этом в следующем файле у тебя уже аккуратней с этим

Reply:
Пройдусь по названиям, поправлю)